erpc_c/port/erpc_port_memmanager.cpp

```diff
@@ -58,13 +58,13 @@ void operator delete[](void *ptr) THROW
 
 void *erpc_malloc(size_t size)
 {
-    void *p = MEM_BufferAllocForever(size, 0);
+    void *p = MEM_BufferAllocWithId(size, 0);
```

Comment:
looking at [[NXP_miniambulance_serial2ble](https://github.com/w562601331/NXP_miniambulance_serial2ble)](https://github.com/w562601331/NXP_miniambulance_serial2ble/blob/8fd71feb01b494f54ad50ccb88b0b17c732f2129/framework/MemManager/Interface/MemManager.h), looks like MEM_BufferAllocWithId should get 3 parameters. Maybe you meant to use MEM_BufferAlloc?